src/select.rs

```diff
@@ -318,10 +318,14 @@ impl OneSelector {
             } else {
                 first_record.len() - 1
             }),
-            OneSelector::Index(i) => {
+            OneSelector::Index(mut i) => {
                 if first_record.is_empty() {
                     return fail!("Input is empty.");
                 }
+                // 9999 is a sentinel value that means "last column".
```

Comment:
It does not look like a good sentinel value. There can be very wide CSV file that contains more than 9999 columns.

Hypothetical script which selects all columns sequentially from a wide input CSV file would mysteriously produce wrong results when the file is wide enough, but the problem won't be reproducible with small test data.

It should be at least `usize::MAX` as a sentinel or better some dedicated keyword (that may map to `usize::MAX` or something else internally).